spec.md

```diff
@@ -304,8 +304,9 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id/las
 
 | Status Code | Description |
 | --- | --- |
-| 200 OK | The expected response body is below. |
-| 410 Gone | Appropriate only for asynchronous delete operations. The platform MUST consider this response a success and remove the resource from its database. The expected response body is `{}`. Returning this while the platform is polling for create or update operations SHOULD be interpreted as an invalid response and the platform SHOULD continue polling. |
+| 200 OK | MUST be returned upon successful processing of this request. The expected response body is below. |
+| 404 Not Found | MUST be returned if the service instance does not exist. |
+| 410 Gone | MAY be returned if the broker knows about the service instance but it had been previously deleted. Appropriate only for asynchronous delete operations. The platform MUST consider this response a success and remove the resource from its database. The expected response body is `{}`. Returning this while the platform is polling for create or update operations SHOULD be interpreted as an invalid response and the platform SHOULD continue polling. While a `410 Gone` is allowed, the RECOMMENDED response is `404 Not Found`. |
```

Comment:
this feels like we're letting an impl detail bleed into the spec and, sort of, reinventing http response codes in the process.  Can we discuss this on the next call?